templates/ndmg_cloud_participant.json

```diff
@@ -9,13 +9,16 @@
             "/data/raw",
             "/data/outputs",
             "participant",
+            "<MODE>",
```

Comment:
analysis level should be optional - valid bids apps only have the first 3 args required.

Reply:
again, added default flag